dpppt-config-backend/src/main/java/org/dpppt/switzerland/backend/sdk/config/ws/model/GAENSDKConfig.java

```diff
@@ -10,12 +10,28 @@
 
 package org.dpppt.switzerland.backend.sdk.config.ws.model;
 
+import ch.ubique.openapi.docannotations.Documentation;
+
 public class GAENSDKConfig {
 
+	@Documentation(description = "Lower threshold that is sent to the GAEN to count encounters with other devices. It" +
+			" is expressed in dB attenuation from the Bluetooth chip.",
+			example="53")
 	private Integer lowerThreshold = 53;
+	@Documentation(description = "Higher threshold that is sent to the GAEN to count encounters with other devices It" +
+			" is expressed in dB attenuation from the Bluetooth chip.")
 	private Integer higherThreshold = 60;
+	@Documentation(description = "Multiplication factor used to weigh the return value of the GAEN in " +
+			"attenuationDuration[0]",
+			example = "1.0d")
 	private Double factorLow = 1.0d;
+	@Documentation(description = "Multiplication factor used to weigh the return value of the GAEN in " +
+			"attenuationDuration[1]",
+			example = "0.5d")
 	private Double factorHigh = 0.5d;
+	@Documentation(description = "Minimum duration of exposure in minutes, before an encounter is as " +
```

Comment:
```suggestion
	@Documentation(description = "Minimum duration of exposure in minutes, before an encounter is added as " +
```

Sorry, my suggestion just sounds wrong :D do you have a better suggestion?

Reply:
OK, if I got it right this time, it's the following:
```
Minimum duration of exposure during one day reported by the GAEN before the user 
is alerted of an exposure risk
```